### PR TITLE
HDS-1670-fix-accordions-with-borders

### PR DESCRIPTION
**Pull request by mrTuomoK**
## Description

Add example of stacking Accordions with borders (inside Cards), avoiding the double-border -issue.

## Related Issue

[HDS-1670](https://helsinkisolutionoffice.atlassian.net/browse/HDS-1670)

## How Has This Been Tested?

Local machine

## Screenshots (if appropriate):

image

[HDS-1670]: https://helsinkisolutionoffice.atlassian.net/browse/HDS-1670?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by mikkojamG — APPROVED** [review]
👍

File: site/src/docs/components/accordion/code.mdx [hunk 1]
```diff
@@ -90,6 +90,28 @@ import { Accordion } from 'hds-react';
...
+  <Accordion size="s" card border heading="How to publish data?" language="en" style={{ maxWidth: '360px' }}>
+    To publish your data, open your profile settings and click the button 'Publish'.
+  </Accordion>
+  <Accordion size="s" card border heading="How to publish data?" language="en" style={{ maxWidth: '360px', borderTop: 0 }}>
+    To publish your data, open your profile settings and click the button 'Publish'.
+  </Accordion>
+  <Accordion size="s" card border heading="How to publish data?" language="en" style={{ maxWidth: '360px', borderTop: 0 }}>
+    To publish your data, open your profile settings and click the button 'Publish'.
+  </Accordion>
+</>)
```
Comment (harriplappalainen) [inline site/src/docs/components/accordion/code.mdx:110]: Maybe we should add this to storybook as well?

Comment (harriplappalainen) [reply site/src/docs/components/accordion/code.mdx:110]: Would be great to improve the Accordion Card story by adding headings before each accordion to be more descriptive. Card | Small card with borders | Normal card with borders | Large card with borders ?
Screenshot 2023-05-17 at 9 07 30

**Review by harriplappalainen — COMMENTED** [review]
Nice work! 👍 I was thinking could there be some storybook action as well? Read more in the comments 🤓

File: site/src/docs/components/accordion/code.mdx (hunk 1 above)
Comment (mrTuomoK) [reply site/src/docs/components/accordion/code.mdx:110]: Ah ok, I'll do those too 👍

Comment (mrTuomoK) [reply site/src/docs/components/accordion/code.mdx:110]: > Maybe we should add this to storybook as well?

Yeah... and this is what I don't like, keeping two different documentations up-to-date 😭 😆

Comment (harriplappalainen) [reply site/src/docs/components/accordion/code.mdx:110]: While you're at it, add a proper heading above the accordions to match your other [PR](https://github.com/City-of-Helsinki/helsinki-design-system/pull/1030).

**Pull request merged**